config/clusters/jupyter-health/staging.values.yaml

```diff
@@ -9,5 +9,28 @@ jupyterhub:
         secretName: https-auto-tls
   hub:
     config:
-      GitHubOAuthenticator:
+      JupyterHub:
+        authenticator_class: generic-oauth
```

Comment:
```suggestion
        # Uses CHCS auth provider
        # Note: 2i2c engineers can not log in via this, so they can not provide support that
        # requires logging into this hub. But since Jupyter Health team members have access to this
        # repo, this is acceptable
        authenticator_class: generic-oauth
```

Reply:
My suggestion is we make this comment, and merge this as is, and accept that 2i2c engineers can't provide support that requires logging into the hub.

Reply:
@minrk if this looks good to you, I suggest you accept the suggestion and merge it!